Frontend/src/app/core/authentication-api.service.ts

```diff
@@ -41,5 +42,15 @@ export class AuthenticationApiService {
     return this._http.get(`${this._baseUrl}/auth/logout`, {headers: this._headers, responseType: 'text'});
   }
 
+  register(userDTO: UserDTO): Observable<any>{
+    return this._http.post(`${this._baseUrl}/register`, {
+      firstName: userDTO.firstName,
+      lastName: userDTO.lastName,
+      email: userDTO.email,
+      password: userDTO.password,
+      verified: false
```

Comment:
This can be simplified by extending Serializable in UserDTO class. Example can be found in model folder.